src/components/Footer.tsx

```diff
@@ -49,6 +57,14 @@ const SocialImg = styled.img`
   margin: 8px;
 `
 
+const SocialLinks = styled.div`
+    display: flex;
+    height: 40px;
```

Comment:
I think that just a tiny bit more space after this section would be nice.

```suggestion
    height: 45px;
```